res/database/Default/user/scr0llbaer.xml

```diff
@@ -1764,6 +1784,152 @@ So that's 4120 per episode.</en>
 			<speed min="30" max="80" />
 			<studio_size value="2" />
 		</scripttemplate>
+
+		<scripttemplate product="2" licence_type="3" guid="71f4752c-2c8f-41f0-82c6-3461e02e8160" creator="9551" comment="TODO: To test a future 'franchise script' feature. Episode titles and contents by ChatGPT. A bit formulaic perhaps.">
```

Comment:
There is a template definition pattern that might be of interest here (see also scripttemplate-ron-visiting).
You can define a template with many, many child templates (e.g. 20 or more). The parent's episodes definition is much lower (e.g. 5). When the actual script is created 5 random episodes are chosen. For now, the very first child template is always included (assumption, there is a pilot episode for a series which is defined as first child; because the current intention is that two script instances created from the same template have nothing to do with each other).

Of course the episodes must not depend on each other...

Reply:
I know about the feature but I think episode randomization shouldn't be applied to this kind of (franchise) template, only to the very very generic script templates with a huge variety of variables and combinations.